### ISSUE TITLE
What's happenned to Kanboard?

### ISSUE THREAD

**Issue by acs-ferreira** [kanboard/kanboard#3144]
Latest release is a mess, i can't even recognize my Kan!

I had no other choice than `git reset --hard 8cd45e8`

I really don't like this new "look".

**Comment by fguillot** [comment]
This is really constructive.

Each time something change, there is half of people who like the changes and the rest that complaint because something has changed.

By the way, this post is really worth reading: https://nolanlawson.com/2017/03/05/what-it-feels-like-to-be-an-open-source-maintainer/

**Comment by sailfish009** [comment]
i really like new look. 
previous look so confused to me.

**Comment by acs-ferreira** [comment]
I agree it's not constructive, but IMHO such kind of changes should involve users and i think that their opinion matters, i don't think that implement it with a simple changelog line is the best way...

At least, having an option to switch views is, IMO, the best way to implement things like this.

We are a few using Kanboard, we thank you for your hard work, but none of us are pleased with this timeline.

But OK, you are the maintainer, you have the lead of your project. No problem. :)

**Comment by EstebanMonge** [comment]
Hello I agree that is a project that solves the needs of the maintainer and his clients, yesterday I updated Kanboard and today a lot of people start to cry, my answer is: thats the update!

If the maintainer want to spent time in make your customer and open source community happy, maybe you can use a switch in configuration with New layout or Old layout... a button that can allow the people to conserver the old dashboard...

Or maybe you want to send a pull request with the old layout and the button to switch it, and try to maintain the changes and resolve the problems of maintain two views that make the same thing, etc.

**Comment by NK74** [comment]
> I really don't like this new "look".

Same 👍

**Comment by oliviermaridat** [comment]
To earn this kind of topics, Kanboad must really have reached a new step! Congratulations @fguillot , your "minimalist" project is going really well! For what it worth, I think you have always been wiser than anyone could expect in terms of architecture and features. If you doubt it: you can trust yourself. And nobody has never been forced to install the last version ;-)
 
Please guys, if you really like Kanboard, may I suggest than you propose something that fits your needs and the needs fixed by the new look?

**Comment by jaiume** [comment]
For me the new dashboard slows me down a lot. I have many projects and even just lists of other things I maintain (e.g. My goals for the year).  I also have many 'parked' items where I'm waiting on a third party or some other event to happen. 

Before I could use the dashboard as a 'navigator' or menu to go to each board. Now I have to do a **LOT** of scrolling.

Please have an option to have short list of my active projects. Or even a **collapsed** view of the new look would be fine.

Love Kanboard, my life would be a mess without it.

Thanks
Jamie

**Comment by szawadski** [comment]
I also got few complaints of users about the new dashboard, but I agree with @oliviermaridat, Kanboard is an incredibly rich and well architectured application that satisfies both end users and admins (I have rarely seen such a good acceptation rate amongst teams, from top management to drafter, young or old...). 

I am confident that based on our constructive comments and proposals, the solution will evolve, step by step, and we will get used to it. We will start to find advantages to the new view and will finally love it.

Same for the users management page: as soon as search feature will be available (#1426), it will become much better than the previous table.

Thanks @fguillot for your hard and useful work!

**Comment by erpnedir** [comment]
One of the pros of Kanboard is its simplicity. The problem about the new dashboard is it just gives an opposite feeling right after login.
Other than that, congrats for the excellent product.

**Comment by fguillot** [comment]
To avoid cluttering the bug tracker, there is now a forum for open discussions: https://forum.kanboard.net/

**Comment by sailfish009** [comment]
i couldn't log in forum via github account, 

> Internal Server Error
> 
> The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
